src/app.py

```diff
@@ -107,5 +107,29 @@ def add_course():
     return redirect(url_for("start"))
 
 
+# router to course detailed page
+
+
+@app.route("/course_page", methods=["GET", "POST"])
+def course_page():
+    df = get_df_from_csv_in_s3(s3, bucket_name, mock_data_file)
+    courses = df.loc[0, "courses"]  # For PoC purpose
+    # Parsing it into a Python list
+    courses = ast.literal_eval(courses)
+    return render_template("course_page.html", courses=courses)
+
+
+@app.route("/plan_page", methods=["GET", "POST"])
+def plan_page():
+    return render_template("plan_page.html")
+
+
+@app.route("/profile_page", methods=["GET", "POST"])
+def profile_page():
+    df = get_df_from_csv_in_s3(s3, bucket_name, mock_data_file)
+    username = df.loc[0, "username"]
+    return render_template("profile_page.html", username=username)
```

Comment:
Could you add a function description on top of each function starting with a capital letter? Thanks a lot

Reply:
Second this, going through the routes I can't tell what the difference between the home page and the course page are supposed to be. A few lines before each route as a comment describing what the page will be for would be useful. Another thing to consider, keeping different routes in their own files for better modularization and clarity (don't have to worry about this for POC but maybe after when things begin to grow more).